{"name": "test_loans_section", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'asdf' == 'qwer'\n  - qwer\n  + asdf", "trace": "@pytest.mark.usefixtures(\"log_on_failure\")\n    @pytest.mark.smoke\n    def test_loans_section():  # function continue execution even an assertion is failed\n>       assert \"asdf\" == \"qwer\"\nE       AssertionError: assert 'asdf' == 'qwer'\nE         - qwer\nE         + asdf\n\ntest_j_screenshot_only_on_faliure.py:46: AssertionError"}, "start": 1673697636818, "stop": 1673697636818, "uuid": "6bb42d8b-9ca9-4412-897f-040d689edc6b", "historyId": "b75712fbcba8c477d01a86f0d002a8cc", "testCaseId": "57f209bdae5f784a7c8e4c448388e0ca", "fullName": "test_j_screenshot_only_on_faliure#test_loans_section", "labels": [{"name": "tag", "value": "smoke"}, {"name": "tag", "value": "@pytest.mark.usefixtures('log_on_failure')"}, {"name": "suite", "value": "test_j_screenshot_only_on_faliure"}, {"name": "host", "value": "192.168.1.4"}, {"name": "thread", "value": "33919-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_j_screenshot_only_on_faliure"}]}